packages/ui/package.json

```diff
@@ -8,7 +8,7 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "dev:old": "tsup src/index.ts --watch --dts --format esm,cjs"
```

Comment:
nice

Reply:
Yes, that's what happens when you revert broken commits 😅